app/Libraries/Search/BeatmapsetSearch.php

```diff
@@ -60,6 +60,12 @@ public function getQuery()
                     ->should(['term' => ['_id' => ['value' => $this->params->queryString, 'boost' => 100]]])
                     ->should(QueryHelper::queryString($this->params->queryString, $partialMatchFields, 'or', 1 / count($terms)))
                     ->should(QueryHelper::queryString($this->params->queryString, [], 'and'))
+                    ->should([
+                        'nested' => [
```

Comment:
shouldn't this be also shared with the beatmap specific `$nested` query down below?

Reply:
The scoring and matching of the query string is not the same in the other nested filter

Reply:
would this match tag that isn't part of beatmap specific field match? searching by this and difficulty for example

Reply:
Searching for `potato foo difficulty=mayday` returns beatmapsets with `mayday` difficulty with relevance score according to how the fields in the query match `potato foo`